docs/guides/auth.md

```diff
@@ -302,7 +302,26 @@ $client = new helloworld\GreeterClient(
   new Grpc\BaseStub('localhost:50051', []));
 ...
 
-// Authenticating with Google
+// Authenticate using JWT access token (recommended approach)
+function updateAuthMetadataCallback($context)
+{
+    $authUri = $context->service_url;
+    $methodName = $context->method_name;
+    $auth_credentials = ApplicationDefaultCredentials::getCredentials();
+
+    return $auth_credentials->updateMetadata($metadata = [], $authUri);
+}
+$opts = [
+  'credentials' => Grpc\Credentials::createSsl(file_get_contents('ca.pem'));
+];
+
+$client = new helloworld\GreeterClient('localhost:50051', $opts);
```

Comment:
Thanks much. That's way better. How about we simplify the approach a little more.
- simplified the callback.
- standardize on camel-case.
- Using Grpc\ChannelCredentials as opposed to Grpc\Credentials (I assume that this is the right call now).
- replaced ca.pem with roots.pem which is our standard publicly trusted roots file (etc/roots.pem).
- changed $credentials to $channelCredentials to make things more explicit.

```php
// Authenticate using scopeless credentials (recommended approach)
function updateAuthMetadataCallback($context)
{
    $authCredentials = ApplicationDefaultCredentials::getCredentials();
    return $authCredentials->updateMetadata($metadata = [], $context->authUri);
}
$channelCredentials = Grpc\ChannelCredentials::createComposite(
    Grpc\ChannelCredentials::createSsl(file_get_contents('roots.pem')),
    Grpc\CallCredentials::createFromPlugin('updateAuthMetadataCallback')
);
$opts = [
  'credentials' =>  $channelCredentials
];

$client = new helloworld\GreeterClient('greeter.googleapis.com', $opts);
```

It would actually be great to have a helper for this code (a function like `createDefaultGoogleChannelCreds` that would return the $channelCredentials) and tell people use this since I think that this will be the 80% use case.

Sorry if my example is not correct BTW, I haven't touched PHP for at last 15 years :).